From htejun@gmail.com  Wed May  2 16:40:09 2007
From: Tejun Heo <htejun@gmail.com>
Date: Tue, 01 May 2007 11:00:19 +0200
Subject: DOC: Fix wrong identifier name in Documentation/driver-model/devres.txt
To: Greg KH <greg@kroah.com>
Cc: Jeff Garzik <jeff@garzik.org>, Rolf Eike Beer <eike-kernel@sf-tec.de>,  linux-kernel@vger.kernel.org
Message-ID: <463701A3.5060401@gmail.com>


From: Rolf Eike Beer <eike-kernel@sf-tec.de>

Above and below we talk about my_midlayer_create_something, I assume that is
also meant here.

Signed-off-by: Rolf Eike Beer <eike-kernel@sf-tec.de>
Signed-off-by: Tejun Heo <htejun@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/driver-model/devres.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/driver-model/devres.txt
+++ b/Documentation/driver-model/devres.txt
@@ -182,7 +182,7 @@ For example, you can do something like t
 
 	...
 
-	devres_close_group(dev, my_midlayer_something);
+	devres_close_group(dev, my_midlayer_create_something);
 	return 0;
   }
 
